App/Components/Quiz/QuizResultGraph.js

```diff
@@ -41,8 +41,8 @@ export default class QuizResultGraph extends Component {
                     this.props.QuizMailer()
                 }
             }
-            else if(this.props.quizType==='numerical' || this.props.quizType=="multicorrect"){
-                await kbcResponse.getAllNumericalResponse(this.props.passCode, r["startTime"], r["endTime"] )
+            else if(this.props.quizType==='alphaNumerical' || this.props.quizType=="multicorrect" || this.props.quizType=="numeric"){
+                await kbcResponse.getAllalphaNumericalResponse(this.props.passCode, r["startTime"], r["endTime"] )
```

Comment:
TypeError: kbcResponse.getAllalphaNumericalResponse is not a function. (In 'kbcResponse.getAllalphaNumericalResponse(_this.props.passCode, r["startTime"], r["endTime"])', 'kbcResponse.getAllalphaNumericalResponse' is undefined)

The kbcResponse.getAllalphaNumericalResponse function is not defined anywhere in the codebase.

Reply:
@shreeya13garg Camel Case the function properly also when you define it

Reply:
I fixed this error and the camel case problem in e318167